#Hypothesis Test Exercise 4

In [1]:
import pandas as pd
import numpy as np
import scipy
from scipy import stats

In [2]:
data3=pd.read_csv("Costomer+OrderForm.csv")
data3.head()

Phillippines   Indonesia       Malta       India
0   Error Free  Error Free   Defective  Error Free
1   Error Free  Error Free  Error Free   Defective
2   Error Free   Defective   Defective  Error Free
3   Error Free  Error Free  Error Free  Error Free
4   Error Free  Error Free   Defective  Error Free

In [3]:
from numpy.random.mtrand import chisquare
stats.chi2_contingency([data3["Phillippines"].value_counts(), data3["Indonesia"].value_counts(), data3["Malta"].value_counts(), data3["India"].value_counts()])

Chi2ContingencyResult(statistic=3.8589606858203545, pvalue=0.2771020991233144, dof=3, expected_freq=array([[271.75,  28.25],
       [271.75,  28.25],
       [271.75,  28.25],
       [271.75,  28.25]]))

# from the above test we got p-value is 0.277 which is greater than alpha (0.05)
# hence H0 (Null Hypothesis) is True